recipes/texlive-core/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "20150521" %}
+
+package:
+    name: texlive-core
+    version: {{ version }}
+
+source:
+    fn: texlive-{{ version }}-source.tar.xz
+    url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
+    sha256: ed9bcd7bdce899c3c27c16a8c5c3017c4f09e1d7fd097038351b72497e9d4669
+
+build:
+    number: 0
+    skip: True  # [win or osx]
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - pkg-config
+        - cairo
+        - gmp
+        - icu
+        - mpfr
+        - pixman
+        - freetype 2.5.*
+        - libpng 1.6*
+        - zlib 1.2*
+    run:
+        - cairo
+        - gmp
+        - icu
+        - mpfr
+        - pixman
+        - freetype 2.5.*
+        - libpng 1.6*
+        - zlib 1.2*
+
+test:
+    commands:
+        - pdftex --version
```

Comment:
Maybe add a call to bibtex, and perhaps there is a package manager that we can test as well?